src/Resource/Entry.php

```diff
@@ -23,11 +23,6 @@ class Entry extends LocalizedResource
      */
     protected $fields;
 
-    /**
-     * @var array
-     */
-    protected $resolvedLinks = [];
```

Comment:
Previously entries would keep a locale cache of resolved links, but now the SDK uses a global entry registry so there's no need to keep doing this locally